superset/views/database/views.py

```diff
@@ -156,48 +157,45 @@ def form_post(self, form: CsvToDatabaseForm) -> Response:
         ).name
 
         try:
-            utils.ensure_path_exists(config["UPLOAD_FOLDER"])
+            utils.ensure_path_exists(app.config["UPLOAD_FOLDER"])
             upload_stream_write(form.csv_file.data, uploaded_tmp_file_path)
```

Comment:
Not related to this PR, but I think it may be possible to let pandas read directly from the stream, without writing a tempfile:

```
chunks = pd.read_csv(form.csv_file.data.stream)
```

Reply:
@ktmud I think this is a good idea.

Reply:
Actually @ktmud I opted to add the streaming logic in this PR given it was a fairly small additional change and recent iterations of this PR meant I needed to refactor some of Excel flow.